src/Domain/Service/TaskFactory.php

```diff
@@ -30,11 +32,16 @@ public function createTaskInstance(string $taskName): Task
 
     private function createTask(string $taskName): Task
     {
-        if (! $this->container->has($taskName)) {
+        if ($this->container === null || ! $this->container->has($taskName)) {
```

Comment:
This doesn't make any sense because below is a check if container is NOT null. If container is NULL, what does that mean?

Reply:
It makes sense. It is confusing yes. But it makes sense. I did it to satisfy phpstan. But you we should simplify it, because the container can never be null in our context.

Reply:
ok